rktl_sim/src/simulator/car.py

```diff
@@ -13,24 +13,20 @@
 class Car(object):
     def __init__(self, carID, length, pos, orient):
         self.id = carID
-        self._steering_angle = 0.
-        self._steering_limit = math.pi / 6.
-        self._length = length
-        self._length_r = self._length / 2.
-        self._steering_rate = 2*self._steering_limit / 0.25
+
+        # physical constants
+        self._LENGTH = length
+        self._MAX_SPEED = 1.0
+        self._THROTTLE_TAU = 0.25
+        self._STEERING_THROW = math.pi / 6.
+        self._STEERING_RATE = 2*self._STEERING_THROW / 0.25
```

Comment:
These 5 lines are what I mean. They should all be synced to single rosparam file somewhere that the filters, controllers, and sim will all use.

Reply:
That will done in root launch config and passed through node